tests/test_quotation/test_embeddings.py

```diff
@@ -0,0 +1,83 @@
+"""
+Tests for sentence embedding functionality.
+"""
+
+import importlib
+from unittest.mock import Mock, patch
+
+import numpy as np
+import pytest
+
+import remarx.quotation.embeddings
+from remarx.quotation.embeddings import get_sentence_embeddings
+
+
+class TestGetSentenceEmbeddings:
+    @patch("remarx.quotation.embeddings.SentenceTransformer")
+    def test_get_sentence_embeddings_basic(self, mock_transformer_class):
+        """Test basic sentence embedding generation from list of sentences."""
+
+        # Mock the sentence transformer
+        mock_model = Mock()
+        mock_embeddings = np.array([[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]])
+        mock_model.encode.return_value = mock_embeddings
+        mock_transformer_class.return_value = mock_model
+
+        sentences = [
+            "Das ist ein Test.",  # codespell:ignore
+            "Dies ist ein weiterer Test.",  # codespell:ignore
+        ]
+
+        result = get_sentence_embeddings(sentences)
+
+        # Verify the model was initialized with default model name
+        mock_transformer_class.assert_called_once_with(
+            "paraphrase-multilingual-mpnet-base-v2"
+        )
+
+        # Verify encode was called with correct parameters
+        mock_model.encode.assert_called_once_with(
+            sentences,
+            normalize_embeddings=True,
+            show_progress_bar=True,
+        )
+
+        np.testing.assert_array_equal(result, mock_embeddings)
+
+    @patch("remarx.quotation.embeddings.SentenceTransformer")
+    def test_get_sentence_embeddings_custom_model(self, mock_transformer_class):
+        """Test sentence embedding generation with custom model."""
+
+        # Mock the sentence transformer
+        mock_model = Mock()
+        mock_embeddings = np.array([[0.1, 0.2]])
+        mock_model.encode.return_value = mock_embeddings
```

Comment:
Given that there's no validation of this output, just make this a string so that the numpy dependency can be removed from this unit test.

E.g., 
```suggestion
        mock_model.encode.return_value = "embeddings"
```